SwiftUserDefaults.podspec

```
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name = "SwiftUserDefaults"
+  s.version = "0.0.1"
+  s.summary = "A series of Swift friendly utilities for Foundation's UserDefaults class"
+  s.homepage = "https://github.com/cookpad/swift-user-defaults"
+  s.license = { :type => "MIT", :file => "LICENSE.txt" }
+  s.author = { 'liamnichols' => 'liam.nichols.ln@gmail.com' }
```

This is for anybody who maintains the library 🙂 

```suggestion
  s.authors = { 'Liam Nichols' => 'liam.nichols.ln@gmail.com', 'Ryan Paterson' => 'ryan-paterson@cookpad.com' }
```